src/app/MessageDef/MessageDefHelper.cpp

```diff
@@ -31,7 +31,7 @@
 namespace chip {
 namespace app {
 
-#if CHIP_CONFIG_IM_ENABLE_SCHEMA_CHECK && CHIP_DETAIL_LOGGING
+#if CHIP_CONFIG_IM_ENABLE_SCHEMA_CHECK_LOGGING && CHIP_DETAIL_LOGGING
```

Comment:
should we still have CHIP_DETAIL_LOGGING condition?

since the user explicitly states 'please log IM schema' why do we also condition on detail logging?

Reply:
Removed, done, thanks